packages/sync/src/modules/class-full-sync-immediately.php

```diff
@@ -0,0 +1,395 @@
+<?php
+/**
+ * Full sync module.
+ *
+ * @package automattic/jetpack-sync
+ */
+
+namespace Automattic\Jetpack\Sync\Modules;
+
+use Automattic\Jetpack\Sync\Defaults;
+use Automattic\Jetpack\Sync\Lock;
+use Automattic\Jetpack\Sync\Modules;
+use Automattic\Jetpack\Sync\Settings;
+
+/**
+ * This class does a full resync of the database by
+ * sending an outbound action for every single object
+ * that we care about.
+ */
+class Full_Sync_Immediately extends Module {
```

Comment:
Totally optional and opinionated: I must admit the naming of this class pokes in my eye a little bit 😉 Maybe `Immediate_Full_Sync` or `Full_Sync_Immediate` could make more sense?

Reply:
🤷‍♀ I hope it is going to be called Full_Sync soon. and in any case I blame @gravityrail 😛